code/modules/mob/living/simple_animal/abnormality/he/drifting_fox.dm

```diff
@@ -0,0 +1,175 @@
+	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) <= 60)
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/BreachEffect(mob/living/carbon/human/user)
+	..()
+	playsound(src, 'sound/abnormalities/porccubus/head_explode_laugh.ogg', 50, FALSE, 4) // has placeholder
+	icon_living = "fox_breach"
+	icon_state = icon_living
+	pixel_y = -6
+
+//mob/living/simple_animal/hostile/abnormality/drifting_fox/Life()
+	//. = ..()
+	//if(!.) // Dead
+	//	return FALSE
+	//if(health >= 900)
+	//	var/X = pick(GLOB.department_centers)
+	//	var/turf/T = get_turf(X)
+	//	new /mob/living/simple_animal/hostile/umbrella(T)
+
+/mob/living/simple_animal/hostile/umbrella
+	name = "Umbrella"
+	desc = "A tattered and worn umbrella; The fox seems to have many to spare."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "foxbrella" // Gotta Sprite this.
+	icon_living = "foxbrella" // Gotta make Sprites.
+	faction = list("hostile")
+	maxHealth = 125
+	health = 125
+	damage_coeff = list (RED_DAMAGE = 1, WHITE_DAMAGE = 0.7, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 2)
+	move_to_delay = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+	melee_damage_type = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	L.apply_status_effect(/datum/status_effect/umbrella_black_debuff)
+	attack_sound = 'sound/abnormalities/driftingfox/fox_aoe_sound.ogg'
+	attack_verb_continuous = "slashes"
+	attack_verb_simple = "cut"
+	robust_searching = TRUE
+	del_on_death = FALSE
+
+/mob/living/simple_animal/hostile/umbrella/AttackingTarget(atom/attacked_target)
+	. = ..()
+	if(isliving(target) && !ishuman(target))
+		var/mob/living/L = target
+		if(L.stat == DEAD)
+			return FALSE
+	return ..()
+	//umbrella debuff stuff
+/datum/status_effect/umbrella_black_debuff
+	id = "umbrella_black_debuff"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 15 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/umbrella_black_debuff
+
+/datum/status_effect/umbrella_black_debuff/on_apply()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.physiology.black_mod /= 1.3
+		return
+	var/mob/living/simple_animal/M = owner
+	if(M.damage_coeff[BLACK_DAMAGE] <= 0)
+		qdel(src)
+		return
+	M.damage_coeff[BLACK_DAMAGE] += 0.3
+
+/datum/status_effect/umbrella_black_debuff/on_remove()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.physiology.black_mod *= 1.3
+		return
+	var/mob/living/simple_animal/M = owner
+	M.damage_coeff[BLACK_DAMAGE] -= 0.3
+
+/atom/movable/screen/alert/status_effect/umbrella_black_debuff
+	name = "False Kindness"
+	desc = "Your half hearted attempts at kindness have weakened you to BLACK attacks."
+	icon = 'icons/mob/actions/actions_ability.dmi'
+
+/mob/living/simple_animal/hostile/umbrella/death(gibbed)
+	visible_message("<span class='notice'>[src] falls to the ground with the umbrella closing on itself!</span>")
```

Comment:
also probably should replace this for new spans (if you update the branch to upstream master at least)
```suggestion
	visible_message(span_notice("[src] falls to the ground with the umbrella closing on itself!"))
```